<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterFaceClass</class>
 <widget class="QWidget" name="RegisterFaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册新用户</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#020d2c</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="8,2">
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonStartRecord">
        <property name="styleSheet">
         <string notr="true">background:#162e6c;color:gray</string>
        </property>
        <property name="text">
         <string>开始录入</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxName">
        <property name="styleSheet">
         <string notr="true">background:#162e6c;color:gray</string>
        </property>
        <property name="title">
         <string>请输入姓名</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEditName"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>448</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="buttonClose">
        <property name="styleSheet">
         <string notr="true">background:#162e6c;color:gray</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelVideoStreamRegister">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background:black;color:gray</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>Vider Stream</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
